src/lib/checks/[share_token]/practice/EvaluateAnswer.ts

```
@@ -26,5 +27,65 @@ export async function EvaluateAnswer(_: AuthState, data: PracticeData): Promise<
+    case 'multiple-choice':
+      question = question as MultipleChoice
+      return {
+        type: answer.type,
+        solution: question.answers.filter((a) => a.correct).map((answer) => answer.id),
+        reasoning: question.answers.map((answer, i) => (answer.correct ? `Answer ${i} is correct because...` : `Answer ${i} is false because..`)),
+      }
+
+    case 'open-question':
+      question = question as OpenQuestion
+      //todo: use llm to evaluate open-question answer correctness
+      const degreeOfCorrectness = Math.random()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)